src/etc/inc/util.inc

```diff
@@ -928,12 +928,31 @@ function get_interface_list($only_active = false, $include_dmesg = false)
 * NAME
 *   log_error  - Sends a string to syslog.
 * INPUTS
+*   $error     - string containing the syslog message.
+* RESULT
+*   null
+******/
+function log_error($error)
+{
```

Comment:
how about just calling `log_msg($msg)` ? the internals are the same :)

Reply:
) thanks!! setting LOG_ERR explicitly in case log_msg default will change